hi_IN.lproj/Front.strings

```diff
@@ -0,0 +1,189 @@
+/* 
+  Front.strings
+  Typora
+
+ by plutonium.239.811@gmail.com
+*/
+
+/* Common */
+"OK" = "ठीक है";
+"Cancel" = "रद्द करें";
+"Yes" = "हाँ";
+"No" = "नहीं";
+"Close" = "बंद करे";
+"Select" = "चुनते हैं";
+"Continue" = "जारी रखें";
+"Learn Moreâ€¦" = "और जानें € |";
```

Comment:
Should I correct to `"Learn More…" = "और जानें…";` ?

Reply:
Yes